packages/zowe-explorer/package.json

```diff
@@ -1927,26 +1927,26 @@
     ]
   },
   "scripts": {
-    "build": "gulp build && yarn license && webpack --mode development && cd webviews/edit-attributes && yarn && yarn build",
```

Comment:
We will still need the `cd webviews/edit-attributes && pnpm install . && pnpm build` as part of the `build` script, or the feature won't be usable once packaged.

Reply:
Check comment above on first comment made :)

Reply:
Sorry, that was a mistake on my part - resolved